packages/react-core/src/demos/ComposableMenu/examples/ActionsMenuDemo.tsx

```diff
@@ -1,40 +1,22 @@
 import React from 'react';
-import { MenuToggle, Menu, MenuList, MenuItem, MenuGroup, MenuItemAction, Popper } from '@patternfly/react-core';
+import {
+  MenuToggle,
+  MenuItemAction,
+  Dropdown,
+  DropdownGroup,
+  DropdownList,
+  DropdownItem
+} from '@patternfly/react-core';
 import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
 import ClipboardIcon from '@patternfly/react-icons/dist/esm/icons/clipboard-icon';
 import CodeBranchIcon from '@patternfly/react-icons/dist/esm/icons/code-branch-icon';
 import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
 
-export const ComposableActionsMenu: React.FunctionComponent = () => {
+export const ActionsMenuDemo: React.FunctionComponent = () => {
```

Comment:
Similar to my above comment, it seems like this demo should be using Select instead as well based on the functionality.